scripts/data/Submissions.json

```diff
@@ -3,7 +3,7 @@
     "id": "a12a4180-65aa-42ec-a945-5fd21dec0501",
     "type": "ContestSubmission",
     "url": "https://software.topcoder.com/review/actions/DownloadContestSubmission?uid=123456",
-    "memberId": 40493050,
+    "memberId": "b24d4180-65aa-42ec-a945-5fd21dec0502",
```

Comment:
Change unrelated to #179 - This is for when we need to deploy locally. This is the mock data. The `memberId` data type would be set as long and then prevent string uuids from being entered for other submissions. Hence, changed this to a uuid as well